transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -408,6 +409,11 @@ public final ChannelPromise voidPromise() {
         return pipeline.voidPromise();
     }
 
+    final void autoFlushModified(boolean autoFlush) {
```

Comment:
The reason for using `autoFlush` flag was to avoid doing a config lookup, if the lookup isn't an issue, I can just remove this callback. We can just do `config.getOption(ChannelOption.AUTO_FLUSH)`. We do not need `channel.write(EMPTY_BUFFER)`.